src/protocols/data.ts

```diff
@@ -14,9 +15,29 @@ export interface Protocol {
   gecko_id: string;
   cmcId: string;
   category: string;
-  chains: string[];
+  chains: chains[];
   module: string;
   twitter: string;
+  audit_links: string[];
+  slug: string;
+  tvl: number;
+  chainTvls: {
```

Comment:
This type is not the one used to define the response returned from the API, but instead it's a type that defines the schema used internally to store the basic data. This data is then processed on https://github.com/DefiLlama/defillama-server/blob/master/src/getProtocols.ts, where we add the missing properties such as chainTvls, change_1h...